Source/Widgets/Geocoder/Geocoder.js

```diff
@@ -70,6 +70,11 @@ value: searchText,\
 valueUpdate: "afterkeydown",\
 disable: isSearchInProgress,\
 css: { "cesium-geocoder-input-wide" : keepExpanded || searchText.length > 0 }');
+        textBox.addEventListener('focus', function() {
+            setTimeout(function() {
```

Comment:
JavaScript cargo-culting :) [This StackOverflow answer](http://stackoverflow.com/a/19498477) says "In Chrome the selection via .select() doesn't stick - adding a slight timeout resolves this issue.".